Added line 35 to point new contributor to Contribute to Kubernetes docs link. (#20230)
kubernetes/website: CONTRIBUTING.md

Note that code issues should be filed against the main kubernetes repository, while documentation issues should go in the kubernetes.io repository.

> If you’re just getting started, consider exploring this resource [Contribute to Kubernetes docs](https://kubernetes.io/docs/contribute/).

Reviewer: ```suggestion
For more information, see ["Contribute to Kubernetes docs"](https://kubernetes.io/docs/contribute/).
```
